python/ray/tune/trainable.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, config=None, logger_creator=None):
         self.config = config or {}
         trial_info = self.config.pop(TRIAL_INFO, None)
 
+        self._disable_ipython()
```

Comment:
@richardliaw @ericl 
How/why did this _ever_ get approved?
What is this supposed solve?
This is causing breakage for _anyone_ using notebooks.
Please remove or make optional (i.e. gate with a boolean).